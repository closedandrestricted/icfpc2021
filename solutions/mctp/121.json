{"vertices":[[32,152],[13,163],[19,149],[29,176],[29,171],[11,165],[29,159],[18,150],[19,150],[23,169],[36,145],[32,169],[18,152],[18,153],[31,166],[25,138],[33,151],[29,134],[37,151],[17,168],[31,167],[18,152],[40,149],[46,140],[37,157],[32,169],[44,163],[7,173],[28,132],[49,131],[32,169],[26,152],[27,134],[20,159],[29,152],[38,168],[21,153],[45,162],[61,149],[15,159],[8,169],[20,156],[40,150],[21,175],[11,166],[45,146],[34,169],[38,140],[21,177],[31,176],[24,163],[47,108],[6,175],[20,151],[5,174],[39,134],[16,154],[15,158],[27,134],[27,165],[29,174],[17,153],[26,137],[12,162],[4,178],[23,143],[34,173],[18,151],[10,165],[21,146],[3,178]]}