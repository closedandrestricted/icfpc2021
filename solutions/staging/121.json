{"vertices":[[51,198],[65,179],[50,180],[44,175],[47,146],[58,164],[42,157],[55,148],[61,128],[76,143],[84,137],[78,115],[108,181],[83,117],[108,116],[97,118],[101,103],[110,162],[126,172],[95,109],[100,122],[99,121],[103,145],[110,100],[111,124],[104,121],[115,114],[80,108],[94,97],[105,183],[98,124],[104,78],[115,162],[100,135],[129,102],[112,143],[118,126],[125,82],[118,203],[133,86],[114,144],[133,145],[125,181],[101,151],[110,141],[86,74],[64,85],[137,167],[128,132],[177,119],[118,62],[204,205],[139,67],[107,186],[154,132],[92,208],[201,106],[141,116],[179,185],[163,156],[182,90],[170,112],[171,189],[157,162],[151,132],[179,157],[131,159],[119,134],[155,151],[99,166],[87,133]]}