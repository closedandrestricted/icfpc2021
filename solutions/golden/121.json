{"vertices":[[80,154],[92,134],[100,146],[103,152],[74,168],[83,150],[92,167],[77,166],[54,156],[72,145],[80,116],[63,134],[117,195],[98,107],[77,93],[58,96],[67,83],[104,178],[97,196],[84,106],[87,93],[84,140],[95,160],[75,83],[99,92],[91,93],[85,81],[78,116],[60,146],[83,177],[103,80],[53,88],[106,175],[91,107],[77,64],[106,158],[122,74],[58,67],[70,197],[73,48],[105,114],[127,161],[93,194],[107,100],[118,92],[36,150],[56,133],[151,162],[116,142],[177,156],[44,51],[201,211],[58,34],[80,209],[151,155],[54,218],[203,163],[136,141],[186,182],[177,150],[198,140],[175,145],[186,179],[174,152],[200,132],[170,130],[170,126],[208,100],[197,126],[188,98],[209,68]]}